recipes/qss/bld.bat

```diff
@@ -0,0 +1,16 @@
+@echo on
+
+cmake %SRC_DIR% ^
+  -B build ^
+  -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+  -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+  -DBUILD_SHARED_LIBS=OFF ^
+  -DUNIT_TESTS=OFF ^
+  -DBUILD_SAMPLES=OFF
+if errorlevel 1 exit 1
+
+cmake --build build --parallel --config Release
+if errorlevel 1 exit 1
+
```

Comment:
Is there a `make check` or `ctest` we can run here?

Reply:
I could not find any compiled parts that require cmake upstream. I'm probably missing something. Can you clarify why you need this for Windows and not Linux? Can this package be noarch?

Reply:
@ocefpaf this package should be noarch. I was having issues with the Windows CI build failing and I couldn't figure out why, so far the only thing that fixed it was adding a bld.bat file. I don't understand why though and I was hoping someone with more experience with windows could chime in. I went ahead and deleted the file again for you to see the Windows build failure, perhaps I missed something in the logs. Thank you!

Reply:
Looks like everything it working as noarch. Merging it!